profiling/perfetto-connector/libperfetto-connector.cc

```diff
@@ -0,0 +1,110 @@
+#include "perfetto.h"
+#include <unistd.h>
+#include <fcntl.h>
+extern "C" void kokkosp_init_library(const int loadSeq,
+                                     const uint64_t interfaceVer,
+                                     const uint32_t,
+                                     void*);
+extern "C" void kokkosp_finalize_library();
+extern "C" void kokkosp_begin_parallel_for(const char* name,
+                                           const uint32_t,
+                                           uint64_t*);
+extern "C" void kokkosp_end_parallel_for(const uint64_t);
+extern "C" void kokkosp_begin_parallel_scan(const char* name,
+                                            const uint32_t,
+                                            uint64_t*);
+extern "C" void kokkosp_end_parallel_scan(const uint64_t);
+extern "C" void kokkosp_begin_parallel_reduce(const char* name,
+                                              const uint32_t,
+                                              uint64_t*);
+extern "C" void kokkosp_end_parallel_reduce(const uint64_t);
+extern "C" void kokkosp_push_profile_region(
+    char* regionName);
```

Comment:
Is there a reason that `regionName` isn't a `const char *`?

Reply:
Also, I only looked at libperfetto-connector.cc

Reply:
Purely out of consistency with other tools. It could be const char*

Reply:
There may be some historical C reasons behind it.  Anyway, I'll approve.

Reply:
https://github.com/kokkos/kokkos-tools/wiki/Profiling-Hooks#kokkosp_push_profile_region